bin/eds-init.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+(async function () {
+  const fs = require('fs');
+  const path = require('path');
+  const { getConfig } = require('./_util');
+
+  let packageRootPath;
+  try {
+    packageRootPath =
+      path.dirname(require.resolve('@chanzuckerberg/eds')) + '/tokens/json/';
+  } catch (e) {
+    console.error('EDS package not installed. Using local path...');
+    packageRootPath =
+      path.dirname(require.main.path) + '/src/tokens-dist/json/';
+  }
+
+  // read in the config from config file, package json "eds", etc.
+  const config = await getConfig();
+
+  // take the packaged token file and place a copy in the project's 'json' specified path
+  if (config) {
+    try {
+      fs.copyFileSync(
+        packageRootPath + 'theme-base.json',
```

Comment:
where  is theme-base created?

Reply:
`theme-base.json` is generated along side the other style-dictionary output files (e.g., variables.css, et al), and gets shipped with EDS. For consumers it will live **somewhere** in `node_modules/@chanzuckerberg/eds`, and this line will match that location

Each released version of EDS will include an updated version of this file for two uses:

* new projects will start off with a fresh copy that includes all the available tokens
* existing projects can do a diff against their copy and this one to add/remove tokens that changes from EDS version X to Y

That second thing is another reason we have people editing JSON, as it is MUCH easier to build tooling around diffing JSON object as opposed to CSS files or some other content.